src/llama_recipes/utils/train_utils.py

```diff
@@ -207,6 +209,17 @@ def train(model, train_dataloader,eval_dataloader, tokenizer, optimizer, lr_sche
     results["avg_epoch_time"] = avg_epoch_time
     results["avg_checkpoint_time"] = avg_checkpoint_time
 
+    if tracker is not None and track_metrics:
+        tracker.track(avg_train_loss , name='avg_train_loss', context={'subset':'experiment'})
```

Comment:
wrong parameter names? let's make sure we test this to the end of an epoch. we can truncate a dataset so it runs to completion faster.

Reply:
Fixed. Tested and updated the documentation.